modules/installation-aws-upload-custom-rhcos-ami.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-upload-custom-rhcos-ami_{context}"]
+= Uploading a custom {op-system} AMI in AWS
+ 
+If you are deploying to a custom Amazon Web Services (AWS) region, you must
+upload a custom {op-system-first} AMI that belongs to that region.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Create an Amazon S3 bucket with the required IAM
+link:https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role[service role].
+* Upload your {op-system} VMDK file to Amazon S3. The {op-system} VMDK file must
+be the highest version that is less than or equal to the {product-title} version
+you are installing.
+* Download the AWS CLI and install it on your computer. See
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)].
```

Comment:
I think this is fine as is, but the ISG is a bit strict on using parentheses. So I'll pitch another option for this one, but please feel free to take it or leave it: 
Install the AWS CLI Using the Bundled Installer for Linux, macOS, or Unix